### PR TITLE
Update Apollo

### PR DESCRIPTION
**Pull request by pchmelar**
# :pencil: Description
- This PR updates Apollo to 1.0.2

# :bulb: What’s new?
- Apollo recently released a new version 1.0.2 with a naming issue fix, so `pruneGeneratedFiles` is no longer needed

# :no_mouth: What’s missing?
- Nothing

# :books: References
- https://github.com/apollographql/apollo-ios/releases/tag/1.0.2

### CODE REVIEW

**Review by tomas-bat — DISMISSED** [review]

**Review by tommassbrand — DISMISSED** [review]

**Comment by pchmelar** [comment]
Turns out that 1.0.2 doesn't fix the issue (therefore CI check failed). We have to wait for 1.0.3
https://github.com/apollographql/apollo-ios/issues/2546#issuecomment-1289555809

**Review by AdamPenaz — DISMISSED** [review]

**Comment by calvincestari** [comment]
Hi - I wanted to comment on
> Turns out that 1.0.2 doesn't fix the issue (therefore CI check failed). We have to wait for 1.0.3
https://github.com/apollographql/apollo-ios/issues/2546#issuecomment-1289555809

I don't think I'm going to manage to get a fix for the pruning bug into `1.0.3`. Can you manually manage file cleanup until I can get that out?

**Comment by pchmelar** [comment]
> Hi - I wanted to comment on
> 
> > Turns out that 1.0.2 doesn't fix the issue (therefore CI check failed). We have to wait for 1.0.3
> > [apollographql/apollo-ios#2546 (comment)](https://github.com/apollographql/apollo-ios/issues/2546#issuecomment-1289555809)
> 
> I don't think I'm going to manage to get a fix for the pruning bug into `1.0.3`. Can you manually manage file cleanup until I can get that out?

Hey, thanks for stopping by 🙂
No worries, this issue doesn't limit us in any way, as we can still use `pruneGeneratedFiles` until the issue is resolved.

File: ios/DevStack.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 1]
```diff
@@ -14,8 +14,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/apollographql/apollo-ios",
       "state" : {
-        "revision" : "33065d3b3c97fc35ac42bf28f826cac6c61c779e",
-        "version" : "1.0.0"
+        "revision" : "d884d6b7610798fb697a4697423c956f44816103",
+        "version" : "1.0.2"
```
Comment (wooodenleg) [inline ios/DevStack.xcworkspace/xcshareddata/swiftpm/Package.resolved:18]:
```suggestion
        "version" : "1.2.2"
```

Jen test, sorry za ping

**Comment by pchmelar** [comment]
Fixed in 1.0.6 🚀

**Pull request merged**